### ISSUE TITLE
🐛 Bug Report: No null values on optional numeric attributes (rules)

### ISSUE THREAD

**Issue by s3ppo** [appwrite/appwrite#2127]
### 👟 Reproduction steps

create a collection with an optional numeric attribute via the appwrite console.
after updating the rule, the default value is set to 0.. and its not possible to remove it or set it to null because its a number input field

### 👍 Expected behavior

null should be possible

### 👎 Actual Behavior

null is not possible

### 🎲 Appwrite version

Version 0.10.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
This still looks like it's a problem. I created the following attribute:

```json
{
  "key": "int",
  "type": "integer",
  "status": "available",
  "required": false,
  "array": false,
  "min": -9223372036854776000,
  "max": 9223372036854776000,
  "default": null
}
```

`default` is correctly set to `null`, but after the attribute was added, the existing documents had the value set to 0. New document created via the Admin Console passed `null` as the value for the attribute, but the document was still created with 0 as the value. Even when creating a document with an SDK and omitting the attribute, the document is still created with attribute value 0.
